communities/UNAM-Engineering/community-assessment.md

```diff
@@ -0,0 +1,162 @@
+# UNAM Engineering 
+[UNAM Engineering](https://github.com/unamfi) is a small Github organization started on 2016 with the objective to provide students a space
+where they were able to share the projects they have been developing on their classes and hackathons, tools and tips for other students. We
+promote students to go to meetups and various event, help them prepare for job/internship opportunities and provide office hours.
+
+### Definition
+
+- How would you define your community? Try to be open minded, but here are some examples of ways you could define your community: 
+  - Students interested in developing projects in their free time.
+  - Students already working on research and/or developing apps.
+  - Students interested to attend hackathons and meetups.
+  - Students interested in landing internship opportunities.
+  - Students interested in meeting industry companies and people.
+
```

Comment:
We would love to have the university (Teachers & Research) to join since they also have valuable projects that could help inspire students.